distributed-commandbus-jgroups/src/test/java/org/axonframework/jgroups/commandhandling/JGroupsConnectorTest.java

```diff
@@ -317,7 +329,13 @@ public void testUnserializableResponseConvertedToNull() throws Exception {
         when(serializer.serialize(successResponse, byte[].class)).thenThrow(new SerializationException("cannot serialize success"));
         when(serializer.serialize(failureResponse, byte[].class)).thenThrow(new SerializationException("cannot serialize failure"));
 
-        connector1 = new JGroupsConnector(mockCommandBus1, channel1, clusterName, serializer, routingStrategy);
+        connector1 = JGroupsConnector.builder()
+                                     .localSegment(mockCommandBus1)
+                                     .channel(channel1)
+                                     .clusterName(clusterName)
+                                     .serializer(serializer)
+                                     .routingStrategy(routingStrategy)
+                                     .build();
```

Comment:
I favor of time, I haven't done that ^_^;
I do agree this would be valuable, but I'd mark the tests as a 3, whilst having the builders is definitely a 1. Might justify a future ticket.